projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
 **Refer to the endpoints table above for *deleting* a single booking**
 
 ## Testing
-You should be testing the correctness of your code throughout your project. Each JavaScript class file in your project should have its own test file.
 
-Your testing suite should test:
+You should NOT use the original data files in the data directory for testing. These are big files, to begin with, and a real-world dataset would have millions of records. That's far too big to use every time you want to run a test.
 
-* Class properties
-* Class methods
+Instead, you should create small, sample datasets that match the structure of the application data and use these for your test data. By creating this sample dataset, you will also know if your functions are working correctly because you can do the calculations by hand with a much smaller dataset.
 
-Remember to test all possible outcomes (happy/sad/etc).  Ask yourself:  
-  - What is the value of each property?  
-  - Does the method return anything?  
-  - Does the method update any properties?
-  - Are there different possible outcomes to test for based on different arguments being passed in?
+You are expected to test:
 
-You are *not expected* to test:
+All functions that do not update the DOM. This means everything in your scripts.js file should be tested.
+Remember to test all possible outcomes (happy/sad/etc). Ask yourself:
 
-* DOM manipulation / DOM manipulating methods (like `document.querySelector(...)`)
-* Fetch calls
+Does the function return anything?
+Are there different possible outcomes to test for based on different arguments being passed in?
+You are not expected to test:
 
-## Workflow
-You will want to submit PRs to your accountabilibuddy to:
+DOM manipulation / DOM manipulating function (like document.querySelector(...))
+Fetch calls
 
-* You must give your accountabilibuddy collaboration access to your repo.
-* You must submit *at least* 2 PRs to your accountabilibuddy for review.
-* You must wait for your accountabilibuddy to review your PRs, and allow THEM to merge any PRs you submit.
+## Accessibility
 
-It is up to you to decide what changes warrant a PR – remember we want to submit PRs that have significant changes and potential for feedback. Think about what functionality you’re struggling with or have questions about or need help with. As an accountabilibuddy, you are responsible for reviewing at least 2 PRs from your partner.
```

Comment:
Do we want to get rid of the accountabilibuddy peer review?

Reply:
It's still required, its just moved up to the start of the spec so that it doesn't get lost in the technical requirements. I always had students miss it even after pointing it out during kickoffs, so I decided to move it. However, I can put it back down here, if you'd prefer.

Reply:
Ah, like the students, I also missed it!   No, thats great.  Thanks for moving it to somewhere more visible for students.  Sorry for my oversight!